README.md

```diff
@@ -38,25 +55,25 @@ This will then set the appropriate server endpoint in `environment.ts`
 
 Run project:
 
-```bash
+```
 yarn start
 ```
 
 Build project
 
-```bash
+```
```

Comment:
Not a huge deal, but it may be worth moving this below `yarn run storybook` and `yarn lint` as it's the least frequently run command (and isn't necessary for the development environment).

Reply:
will do!

Reply:
@tackc let me know if this works